src/pages/Signup.jsx

```diff
@@ -1,7 +1,73 @@
-import React from "react";
-import { Button, Flex, Heading, Text } from "@chakra-ui/react";
+import React, { useState } from "react";
+import {
+  Button,
+  Flex,
+  Heading,
+  Text,
+  FormControl,
+  Input,
+  Center,
+  Alert,
+  AlertIcon,
+  useToast,
+} from "@chakra-ui/react";
+import { TbArrowBadgeRight } from "react-icons/tb";
+import userServices from "../services/users";
 
 const Signup = () => {
+  const [passwordMatch, setPasswordMatch] = useState(false);
+  const toast = useToast();
+  const [signUpData, setSignUpData] = useState({
+    email: "",
+    nickname: "",
+    password: "",
+    repeatPassword: "",
+  });
+
+  const handleInputChange = (e) => {
+    setSignUpData({
+      ...signUpData,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleCreateAccount = async (e) => {
+    e.preventDefault();
+    console.log(signUpData.password, signUpData.repeatPassword);
+    if (signUpData.password !== signUpData.repeatPassword) {
+      setPasswordMatch(true);
+    } else {
+      setPasswordMatch(false);
+      try {
+        const userObject = {
+          email: signUpData.email,
+          nickname: signUpData.nickname,
+          password: signUpData.password,
+        };
+        const response = await userServices.signup(userObject);
+        localStorage.setItem("token", response.token);
+        toast({
+          title: "Account created.",
+          description: "Get ready to start tapping!",
+          status: "success",
+          duration: 4000,
+          isClosable: true,
+        });
+      } catch (error) {
+        {
```

Comment:
You don't need to wrap your toast in { error && } because it is already in a catch block